zone/task_manager.cpp

```diff
@@ -614,6 +642,22 @@ void TaskManager::TaskQuestSetSelector(
 		return;
 	}
 
+	// live prevents mixing selection types (also uses diff opcodes for solo vs shared tasks)
+	// to keep shared task validation live-like (and simple), any shared task will
+	// forward this to shared task validation and non-shared tasks will be dropped
```

Comment:
Will need to make sure this is documented so it's understood mixing shared tasks with other types won't work

Reply:
Made note of this in the PR description